app/ErrorHandlerProvider.php

```diff
@@ -66,6 +82,29 @@ private function createNewApiProblem(Exception $e): ApiProblem
         return $problem;
     }
 
+    /**
+     * Returns a new ApiProblem just like createNewApiProblem(), but also removes some internal debug info pertaining to
+     * CultureFeed from the error message.
+     *
+     * E.g. "event is not known in uitpas URL CALLED: https://acc.uitid.be/uitid/rest/uitpas/cultureevent/..." (etc)
+     * becomes "event is not known in uitpas ".
+     * The trailing space could easily be removed but it's there for backward compatibility with systems that might have
+     * implemented a comparison on the error message when this was introduced in udb3-uitpas-service in the past.
+     */
+    private function createNewApiProblemFromCultureFeedException(Exception $exception): ApiProblem
+    {
+        $apiProblem = $this->createNewApiProblem($exception);
+
+        $title = $apiProblem->getTitle();
+
+        // Remove "URL CALLED" and everything after it.
+        $formattedTitle = preg_replace('/URL CALLED.*/', '', $title);
```

Comment:
Is this covered by a test somewhere?

Reply:
No, it was also not tested before in `udb3-uitpas-service`, because we have no tests for anything in `app` since it's hard to test these controller/service providers.

I could extract it to a separate class and add tests for that, but I feel like that would shift focus from consolidating & archiving as much repositories as possible to adding more test coverage for existing functionality.

I tested it manually though just in case. It's easy to see in action on a URL like `https://io.uitdatabank.dev/uitpas/events/bla`